src/pull-request/reviewers-list.html

```diff
@@ -4,6 +4,7 @@
       :host {
         display:flex;
         flex-wrap: wrap;
+        flex-direction: column;
```

Comment:
I would like this to be `row` instead. This would make the view more compact when possible:
![screenshot from 2016-05-19 11 49 52](https://cloud.githubusercontent.com/assets/5946422/15389552/d349beee-1db7-11e6-8f71-26f966dcfc1b.png)
Versus:
![screenshot from 2016-05-19 11 50 30](https://cloud.githubusercontent.com/assets/5946422/15389573/e602062c-1db7-11e6-8cae-df5b297fd2d2.png)